pkg/xds/envoy/listeners/v3/access_log_configurer.go

```diff
@@ -61,12 +66,18 @@ func convertLoggingBackend(mesh string, trafficDirection envoy.TrafficDirection,
 		}
 		return fileAccessLog(format.String(), cfg.Path)
 	case mesh_proto.LoggingTcpType:
-		cfg := mesh_proto.TcpLoggingBackendConfig{}
-		if err := proto.ToTyped(backend.Conf, &cfg); err != nil {
-			return nil, errors.Wrap(err, "could not parse backend config")
+		if proxy.Metadata.Features.HasFeature(core_xds.FeatureTCPAccessLogViaNamedPipe) {
+			log.Info("kuma-dp has named pipe feature flag")
```

Comment:
Would this at debug level be enough?

Reply:
Not sure if this log is useful. There is no information what is the DP. The only thing you figure out from reading it is that "there is a DP in the system that does not support named pipe log".

If you want to make a note of this, maybe the better place would be when DP is connecting to the CP (`dataplane_status_tracker.go`).

I'm ok leaving it on DEBUG level, but I'd consider removing it.